examples/evm-to-evm-generic-mesage-passing/src/transfer.ts

```diff
@@ -50,6 +73,30 @@ export async function genericMessage(): Promise<void> {
     transferTx as providers.TransactionRequest
   );
   console.log("Sent transfer with hash: ", response.hash);
+
+  console.log("Waiting for relayers to bridge transaction...");
+
+  let i = 0;
```

Comment:
Can we please just extract this to waitUntilBridged (or something similar to make it a bit cleaner).